### DISCUSSION TITLE
How to remove Preview Deployments

### DISCUSSION THREAD

**Discussion by steven-js** [vercel/vercel#5962] [Help] [answered]
Hello,

I would like to remove the three Preview Deployments that are listed at the bottom of my vercel project.

Will this impact my master branch?

When I click on the options, I can't see an option to delete. Should this be done in vercel or github?

Thank you!

<img width="323" alt="Screenshot 2021-03-10 at 18 14 16" src="https://user-images.githubusercontent.com/35059438/110677127-b4b4ee80-81cc-11eb-9b50-ba7b7744fd01.png">

**Comment by strboul** [comment, marked as answer]
I think you can do it via [Vercel CLI](https://vercel.com/docs/cli).

After installing it, it's possible to list all the environments you have and then remove them by URL. 

```bash
vercel ls <project-name>
vercel rm <preview-url>
```

I haven't found a way for bulk deletion but I guess it's possible to do with a bit of bash loop and AWK.

**Reply by steven-js** [reply]
Thank you! Just following the docs and that page and then found the route for the Project Overview page on the Vercel Dashboard :)
